typescript-react/src/components/Presenters/Presenter.module.css

```diff
@@ -7,6 +7,12 @@
   .Presenter {
     padding: 0 12px;
   }
+
+  :global .MobileBody {
```

Comment:
Same as above, but I believe there's a global hook class `.dslPresenter`?

Reply:
Yes, thank you. Wasn't aware that this class was being used as a hook